### DISCUSSION TITLE
cmd at launch

### DISCUSSION THREAD

**Discussion by aloispichler** [kovidgoyal/kitty#6853] [Q&A]
I’d like to see the variables `$KITTY_WINDOW_ID` and `$KITTY_LISTEN_ON` automatically on kitty startup. How can this be accomplished automatically?
Thanks in advance.

**Comment by cybardev** [comment]
Have you tried using `echo` in a `startup_session` file?

If not, try that. Here's the relevant docs: https://sw.kovidgoyal.net/kitty/overview/#startup-sessions
